content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,345 @@
+        },
+        "series": [
+            {
+                "type": "pie",
+                "data": [
+                    ["Indigenious Beliefs", 10.7],
+                    ["Other Christian", 35.3],
+                    {
+                        "name": "Roman Catholicism",
+                        "y": 10.0,
+                        "sliced": True,
+                        "selected": True,
+                    },
+                    ["Islam", 43.5],
+                    ["Others", 0.5],
+                ],
+            }
+        ],
+    }
+)
+```
+The code above plots the pie chart using the following methods:
+- `from browser import window`: imports the window module from, thus allowing Brython access to Javascript object.
+- `plotBackgroundColor`: It plots the background color for the chart area.
+- `plotBorderWidth`: It plots the width of the chart area in pixels.
+- `plotShadow`: This method takes in boolean values and adds a drop shadow to the area. 
+- `renderTo`: This method is like `document.getElementBy` in javascript. It will take the `id` of the `div` tag in our HTML code.
+- `tooltip`: This object contains the details in the small box when we hover on our chart.
+- `allowPointSelect`: If set as `true`, it allows you to toggle or select any of the slices in the pie chart
+- `Series`: this is a set of data. In this case, it is the religious distribution of people in Nigeria.
+
+When we execute this code, our pie chart is as shown below:
+
+![piechart1](engineering-education/data-representation-with-react-and-ant-design/piechart1.png) 
+
+![piechart2](engineering-education/data-representation-with-react-and-ant-design/piechart2.png)
+
+#### Creating a scatterplot using Brython.
+A scatterplot tests the relationship between two factors. To achieve this, we will put in place the HTML file used for the pie chart. The only difference is the code in the chart.py file.
+
+##### Chart.py file
+The `chart.py` file for the scatterplot is as below:
+
+```python
+from browser import window
+
+b_highchart = window.Highcharts.Chart.new
+b_highchart(
+    {
+        "chart": {"type": "scatter", "zoomType": "xy", "renderTo": "container"},
+        "title": {
+            "text": "scatterplot of the height compared to the weight of 53 Individuals by Race."
+        },
+        "xAxis": {"title": {"text": "Height (cm)"}},
+        "yAxis": {"title": {"text": "Weight (kg)"}},
+        "plotOptions": {
+            "scatter": {
+                "marker": {
+                    "radius": 5,
+                    "states": {
+                        "hover": {"enabled": True, "lineColor": "rgb(100,100,100)"}
+                    },
+                },
+                "states": {"hover": {"marker": {"enabled": False}}},
+                "tooltip": {
+                    "headerFormat": "<b>{series.name}</b><br>",
+                    "pointFormat": "{point.x} cm, {point.y} kg",
+                },
+            }
+        },
+        "series": [
```

Comment:
You can add the function of this `series` attribute when explaining this code snippet.

Reply:
I am trying to avoid repetitions. I already explained it in the piechart section